src/InterNations/Component/Solr/Expression/ExpressionBuilder.php

```diff
@@ -139,12 +139,13 @@ public function fzz($expr, $similarity = null)
      *
      * @param string $start
      * @param string $end
-     * @param boolean $inclusive
+     * @param bool $inclusiveFrom
+     * @param bool $inclusiveTo
      * @return ExpressionInterface
      */
-    public function range($start = null, $end = null, $inclusive = true)
+    public function range($start = null, $end = null, $inclusiveFrom = true, $inclusiveTo = false)
```

Comment:
Shouldn't it be ``$inclusiveTo = true`` not to break current ``$inclusive = true`` behaviour?